manifests/c/chrisant996/Clink/1.4.21/chrisant996.Clink.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: chrisant996.Clink
+PackageVersion: 1.4.21
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ProductCode: clink_1.4.20.236aba

```

Comment:
```suggestion
ProductCode: clink_chrisant996
```

Reply:
Ah, interesting, thanks @Trenly, I didn't realize wingetcreate wouldn't know to update the product code.  Hopefully this is just a one-time change that needs to happen!